Municipality/tygron.mod2g

```diff
@@ -8,23 +8,21 @@ exit=never.
 %Order of the module is standard linear.
 module tygron {
 
+	if goal(answered_request("PERMIT", RequestID, AnswerID)) , bel(open_request("PERMIT", RequestID, ContentLinkID, SenderID, ActionlogIDs, Price, Multipolygon, AreaSize, AnswerList)) 
+		then
+		popup_answer_with_date(RequestID, AnswerID, 0) + 
+		delete(open_request("PERMIT", RequestID, ContentLinkID, SenderID, ActionlogIDs, Price, Multipolygon, AreaSize, AnswerList)) + 
+		insert(answered_request("PERMIT", RequestID, AnswerID)).
+
 	% If there is a goal to answer a request, answer the request.
-	if goal(answered_request(RequestID, AnswerID)) , bel(open_request(RequestType, RequestID, ContentLinkID, SenderID, ActionlogIDs, Price, Multipolygon, AreaSize, AnswerList))then popup_answer(RequestID, AnswerID)
-		 + delete(open_request(RequestType, RequestID, ContentLinkID, SenderID, ActionlogIDs, Price, Multipolygon, AreaSize, AnswerList)).
+	if goal(answered_request(RequestType, RequestID, AnswerID)) , bel(open_request(RequestType, RequestID, ContentLinkID, SenderID, ActionlogIDs, Price, Multipolygon, AreaSize, AnswerList))
+		then
+		popup_answer(RequestID, AnswerID) +
+		delete(open_request(RequestType, RequestID, ContentLinkID, SenderID, ActionlogIDs, Price, Multipolygon, AreaSize, AnswerList)) + 
+		insert(answered_request(RequestType, RequestID, AnswerID)).
 	
 	% If there is a goal to have a park, build a park.
 	% Add budget restriction rule, don't build a park when low on money!
 	if goal(buildPark(ZoneID,MultiPolygon)), bel(functions(FunctionList), member(['stadspark',FunctionID,List],FunctionList)) 
 		then building_plan_construction(FunctionID, 1, MultiPolygon).
-	
-	% If there is a goal to have an AZC, build an AZC.
-	if goal(buildAZC), bel(functions(FunctionList), member(['azc',FunctionID,List],FunctionList)) 
-		then building_plan_construction(FunctionID, 1, multipolygon('MULTIPOLYGON (((10 20, 10 30, 210 30, 210 20, 10 20)))')).
-	
-	% If there is a function to build a road
-	% Then use building_plan_construction to build it (always 1 floor)
-	if bel(functions(FunctionList), member([_,FunctionID,List],FunctionList), member('ROAD',List)) 
```

Comment:
Building an AZC at a fixed polygon is not functional to me I don't see a problem in leaving it out.